src/.jshintrc

```diff
@@ -12,6 +12,9 @@
 
 	"sub": true,
 
+	// Support: IE < 10, Android < 4.1
+	// The above browsers are failing a lot of tests in the ES5
+	// test suite at http://test262.ecmascript.org.
```

Comment:
Damn, that's ugly. It also appears to be both undocumented and untested in JSHint (albeit very [longstanding](https://github.com/jshint/jshint/commit/b0afb93ca05493e5994bd908f54bbdf21f4e1f0e)), and just feels like bad form. On the other hand, dummy keys aren't really any better. Color me ambivalent about the change, and—per usual—frustrated with JSON.